src/routes/operations.ts

```diff
@@ -9,6 +9,9 @@ export const index = (operations: CollectionReference) => async ctx => {
       id: op.id,
       creator_uid: op.get("creator_uid"),
       op_code: op.get("op_code"),
+      // This string conversion would actually be done automatically,
+      // but let's be explicit about what's happening.
+      created_at: (op.get("created_at") as Date).toString(),
```

Comment:
P1 - They recommend using a firestore `Timestamp` instead of date https://firebase.google.com/support/release-notes/js#4.13.0. Might be best practice to use `firebase.firestore.FieldValue.serverTimestamp()`

Reply:
Ah interesting. Struggling to figure out how to get the admin sdk firestore client to return `Timestamp` objects - posted on StackOverflow about this. The `firebase.firestore.FieldValue.serverTimestamp` method you mentioned is actually used as a placeholder to tell the Firestore server to set the field value when you save the object. We're getting the object here, and by default the admin sdk seems to return Date objects.

Reply:
This is the syntax. https://stackoverflow.com/questions/43567312/how-do-i-get-the-server-timestamp-in-cloud-functions-for-firebase

We should at least definitely not forget this, because Firebase straight up tells us this is liable to break soon.